src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getDaoInstance(BookDao.class);
+
+        Book book = new Book();
+        book.setTitle("NewTitle");
+        book.setPrice(BigDecimal.valueOf(40));
+        Book bookUpdated = bookDao.create(book);
```

Comment:
Ensure that the 'create' method in 'BookDao' uses 'Statement.RETURN_GENERATED_KEYS' to retrieve and set the generated id for the book. This is crucial for the subsequent findById operation to work correctly.